sentry-rails/spec/fixtures/test_rails_app/app.rb

```diff
@@ -24,15 +25,15 @@ class TestApp < Rails::Application
 
 case Gem::Version.new(Rails.version)
 when -> (v) { v < v5_2 }
-  require "support/test_rails_app/apps/5-0"
+  require "fixtures/test_rails_app/apps/5-0"
```

Comment:
Fixtures reads to me as [Rails fixtures](https://api.rubyonrails.org/v7.0.2/classes/ActiveRecord/FixtureSet.html). How about using `/dummy`? I think that's the standard across Rails gems.

Reply:
`dummy` is a better name, thanks 👍